### PR TITLE
RPM packaging: update for EL8/EL9

### PR DESCRIPTION
**Pull request by space88man**
Update RPM packaging for EL8/EL9
- add example mock chroots
- update for nftables, iptables deprecation in EL9
- EL8: use newer GCC

### CODE REVIEW

**Comment by rfuchs** [comment]
@mika Any opinions on this?

File: el/rtpengine.spec [hunk 1]
```diff
@@ -236,15 +249,25 @@ true
 %attr(0750,%{name},%{name}) %dir %{_var}/spool/%{binname}
 # Documentation
 %doc LICENSE README.md debian/changelog debian/copyright
+%{_mandir}/man8/%{binname}.8*
 
 %files kernel
-/%{_lib}/xtables/libxt_RTPENGINE.so
+## /usr/lib64/xtables/libxt_RTPENGINE.so
```
Comment (mika) [inline el/rtpengine.spec:255]: Hm, what's the idea behind this change?

**Review by mika — COMMENTED** [review]
LGTM overall (unverified/tested though), only one comment about `libxt_RTPENGINE.so`

File: el/rtpengine.spec (hunk 1 above)
Comment (space88man) [reply el/rtpengine.spec:255]: That artifact  was removed in eea3824725

Comment (mika) [reply el/rtpengine.spec:255]: Right, but shouldn't we _drop_ it then here? :)

Comment (space88man) [reply el/rtpengine.spec]: Done and pushed

**Review by mika — APPROVED** [review]
Untested, but LGTM, thanks @space88man

**Pull request closed without merging**